apps/bfd-server/dev/api-changelog.md

```diff
@@ -704,3 +704,8 @@ Future updates may add `Coding.display` values for additional fields.
 	* The "FIXME this should be mapped as an extension valueIdentifier instead of as a valueCodeableConcept" issues were addressed by creating a new common method for adding identifiers to an extension instead of a codeable concept for these fields. The new method is called addExtensionValueIdentifier in TransformerUtils.
 	* The "FIXME: check if this field is non-nullable and if not remove the 'if' check" issues were addressed by comparing the fields to their definition in the rif-layout-and-fhir-mapping.xlsx file. Most fields were found to be non-nullable and so the "if" check was removed.
  
+## BFD-786 Fix Tax Number Mapping in V1 & V2
+
+* Following FHIR mapping changes were made:
+
+	* Tax number has backwards compatibility with current V1 and V2 responses 
```

Comment:
I think this needs more detail. Is there an example that can be shown? @bfausett77 what do you think?

Reply:
I'm grateful

Reply:
And how do I meet God, I need to talk to him, I'm always out of URL" his token, while he URL" also created me 😰😰😰😥